libs/ui/inputs/src/lib/check-toggle/check-toggle.component.html

```diff
@@ -0,0 +1,18 @@
+<label class="inline-flex relative items-start cursor-pointer" [title]="title">
+  <span class="shrink-0">
```

Comment:
In flowbite the span does not nest other elements.
Where did you take your exemple from ?

Reply:
I took the first flowbite example as a base, but had to adapt this part a little bit as the example button breaks (especially with long labels) on mobile devices.